worldedit-cli/src/main/java/com/sk89q/worldedit/cli/CLIPlatform.java

```diff
@@ -160,6 +163,11 @@ public String getPlatformVersion() {
         return ImmutableSet.of();
     }
 
+    @Override
+    public @NotNull RelighterFactory getRelighterFactory() {
+        return (_a, _b, _c) -> NullRelighter.INSTANCE;
+    }
```

Comment:
Why doesn't CLI need a relighter?

Reply:
I honestly don't even know how this is handled by WE, as CLI doesn't have a light engine available (I guess?). NMSRelighter should work there tho...

Reply:
yeah if nvm actually. If there's no server there's no tuinity.